MVC_Activity01/MVC_Activity01/Data/MockDb.cs

```diff
@@ -10,22 +10,60 @@ public static class MockDb
     {
         public static List<Order> Orders = new List<Order>
         {
-            new Order
-            {
-                Id = 1,
-                No = "0001",
-                DocDate = DateTime.Today,
-                CustomerName = "Juan",
-                Amount = 1000
-            },
-            new Order
-            {
-                Id = 2,
-                No = "0002",
-                DocDate = DateTime.Today,
-                CustomerName = "Pedro",
-                Amount = 1500
-            }
+            //new Order
+            //{
+            //    Id = 1,
```

Comment:
Would be good to use only comment `//` for descriptive purposes like what is the purpose of this method.
Unused codes should be removed.

Reply:
ok sir @rjs123431 noted sir.